tests/sentry/sentry_metrics/test_ratelimiters.py

```diff
@@ -89,5 +95,53 @@ def test_writes_limiter_global_limit(set_sentry_option):
         }
     )
 
-    with writes_limiter.check_write_limits(UseCaseKey.PERFORMANCE, key_collection) as state:
+    with writes_limiter.check_write_limits(
+        UseCaseKey.PERFORMANCE, PERFORMANCE_PG_NAMESPACE, key_collection
+    ) as state:
         assert len(state.dropped_strings) == 2
+
+
+def test_writes_limiter_respects_namespaces(set_sentry_option):
+    """
+    Rate limiters can have the same use case but different namespaces,
+    so each namespace should have it's own quota.
```

Comment:
This doesn't seem right to me. Why can't the quota be shared for all backends with the same use case ID? This way we will not have to update all the options in 2 places every time we want to rate limit an org. Then keeping them in sync would be a pain.

Reply:
> This way we will not have to update all the options in 2 places every time we want to rate limit an org.

It uses the same options for both namespaces, the consuming of the quotas is tracked via the namespace, at least that's how im understanding this. 

maybe "each namespace should have it's own quota" should say "each namespace tracks it's own quota", does that make sense?

Reply:
I guess I don't get why prefix_override is passed via quotas into the ratelimiter. It seems a overly circuitous. As far as I can tell it can just be passed to the rate limiter directly. But I might be missing something.